readthedocs/doc_builder/python_environments.py

```diff
@@ -87,6 +98,70 @@ def venv_bin(self, filename=None):
             parts.append(filename)
         return os.path.join(*parts)
 
+    def environment_json_path(self):
+        """Return the path to the ``environment.json`` file for this venv."""
+        return os.path.join(
+            self.venv_path(),
+            'environment.json',
```

Comment:
I do wonder if this should be namespaced a bit more (`readthedocs-environment.json`?) so that it doesn't conflict with anything in the future.

Reply:
Agree.